tests/d1suned_test.py

```
@@ -59,11 +58,57 @@ def test_write(self, tmpdir, file):
+        assert len(irrfile.irr_schedules[0].times) == 2
+        assert irrfile.nsc == 2
+        assert irrfile.irr_schedules[0].times[0] == irrfile.irr_schedules[0].times[1]
+        assert irrfile.irr_schedules[-1].times[0] != irrfile.irr_schedules[-1].times[1]
+
     def test_get_daughters(self):
         with as_file(RESOURCES.joinpath("irr_test")) as inp:
             irrfile = IrradiationFile.from_text(inp)
         daughters = irrfile.get_daughters()
-        assert daughters == ["24051", "25054", "26055", "26059", "27062", "27062900"]
+        expected = ["24051", "25054", "26055", "26059", "27062", "27062900"]
+        for one, other in zip(daughters, expected):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)